README.md

```diff
@@ -147,3 +151,20 @@ Help us make Deltakit better! Check out [Contributor guide](CONTRIBUTING.md)
 
 ## License
 This project is distributed under the [Apache 2.0 License](LICENSE).
+
+
+## Citation
+
+If you find this toolkit useful, please consider citing it:
+
+```bibtex
```

Comment:
Apparently, `software` is recognized by BibLaTeX and not plain BibTeX, but "biblatex" is not a standard fence, so this is fine.